Makefile

```diff
@@ -14,6 +14,10 @@ prepare-test-cluster:
 test-e2e:
 	. ./scripts/run_e2e_tests.sh
 
+.PHONY: test-e2e-on-installed-operator
```

Comment:
```
.PHONY: run-e2e-on-installed-operator
run-e2e-on-installed-operator:
	CGO_ENABLED=0 SKIP_OPERATOR_DEPLOYMENT=true operator-sdk test local ./test/e2e --verbose --no-setup
```

We ca run it directly through a make target. I am not sure about the requirement of the script ```run_e2e_tests_on_installed_operator.sh``` that you have written.

Reply:
We need the script to generate the kubeconfig for operator-sdk to consume.